### ISSUE TITLE
-s's 3 minute limit is abitrary. Allow optional value

### ISSUE THREAD

**Issue by joejulian** [openntpd-portable/openntpd-portable#11]
In the -s option to "Set  the time immediately at startup if the local clock is off by more than 180 seconds" the 180 seconds is arbitrary and does not suit every use case. Please allow that to be overridden, perhaps as "-s [seconds]".

**Comment by busterb** [comment]
What do you mean? If you say -s, it sets the time no matter how distant the local clock.

```
# ntpd -ds
ntp_adjtime returns frequency of -35.154037ppm
ntp engine ready
reply from 96.44.142.5: offset 3614.076439 delay 0.017671, next query 8s 
set local clock to Thu Mar 12 03:09:57 CDT 2015 (offset 3614.076439s)
reply from 132.163.4.101: offset 1807.036920 delay 3614.142648, next query 7s 
reply from 23.92.26.57: offset 1807.027307 delay 3614.153943, next query 6s 
reply from 66.228.42.59: offset 1807.049050 delay 3614.164064, next query 7s 
```

The documentation doesn't mention anything about an arbitary limit where it will not set the time, where did this 180 second quote come from?

```
     -s          Try to set the time immediately at startup, as opposed to
                 slowly adjusting the clock.  ntpd will stay in the foreground
                 for up to 15 seconds waiting for one of the configured NTP
                 servers to reply.
```

**Comment by busterb** [comment]
Ah - this was fixed in 2008 with OpenBSD 4.4. It should be fixed already in the current OpenNTPD release.

https://github.com/openntpd-portable/openntpd-openbsd/commit/524aa6f9d7052866a5c0e002402c45832d272a03
